sql/core/src/test/resources/sql-tests/analyzer-results/sql-udf.sql.out

```diff
@@ -4336,6 +4336,76 @@ RESET spark.sql.ansi.enabled
 ResetCommand spark.sql.ansi.enabled
 
 
+-- !query
+CREATE FUNCTION foo3_14a() RETURNS INT RETURN 1
+-- !query analysis
+org.apache.spark.sql.catalyst.analysis.FunctionAlreadyExistsException
+{
+  "errorClass" : "ROUTINE_ALREADY_EXISTS",
```

Comment:
there is something wrong there. The analysis output is different from the actual output: https://github.com/apache/spark/pull/51427/files#diff-76c104cd92ea47316d13761146c5f255b619348dd81ef23f4c4b343aa0f52258R3899

Reply:
It's likely because we don't drop the functions at the end. So when run the analysis tests, the functions are already there.

Reply:
Make sense. Let me fix this in a separate PR.